packages/apollo-graphql/src/operationId.ts

```diff
@@ -43,26 +42,38 @@
 // the server no longer needs to parse the signature or run its own signature
 // algorithm on it, and the details of the signature algorithm are now up to the
 // reporting agent.
-
 import { DocumentNode } from "graphql";
+import { createHash } from "apollo-env";
 import {
   printWithReducedWhitespace,
   dropUnusedDefinitions,
-  removeAliases,
   sortAST,
-  hideLiterals
+  hideStringAndNumericLiterals
 } from "./transforms";
 
-// The default signature function consists of removing unused definitions
-// and whitespace.
-// XXX consider caching somehow
+// The default signature function consists of removing extra whitespace,
+// sorting the AST in a deterministic manner, hiding string and numeric literals,
+// and removing unused definitions.
+//
+// XXX
+// The hiding of literals is currently being discussed. This behavior
+// should not be depended on in the future, as it's likely we will choose not
+// to hide them at all. The rationale being, if queries are being shipped to
+// a client bundle, exposing PII via this signature is a very small concern,
+// relatively speaking.
 export function defaultEngineReportingSignature(
```

Comment:
Can we just not change defaultEngineReportingSignature at all in this PR, especially since it's an interim quick fix?

Reply:
Concur!

Reply:
Resolved in 7714ee1b9517b7f25701b66d84839bdedb1041fa